src/BenchmarkDotNet.Core/Portability/StringExtensions.cs

```diff
@@ -13,5 +13,10 @@ internal static bool EqualsWithIgnoreCase(this string left, string right)
             return left.Equals(right, StringComparison.OrdinalIgnoreCase);
 #endif
         }
+
+        internal static bool ContainsWithIgnoreCase(this string text, string word)
+        {
+            return text.IndexOf(word, StringComparison.OrdinalIgnoreCase) >= 0;
+        }
```

Comment:
It would be better to introduce a private property:
```cs
        private static StringComparison IgnoreCaseStringComparison
        {
            get
            {
#if !CORE
                return StringComparison.InvariantCultureIgnoreCase;
#else
// http://stackoverflow.com/questions/14600694/where-has-stringcomparison-invariantcultureignorecase-gone
                return StringComparison.OrdinalIgnoreCase;
#endif
            }
        }
```
and use it in both `EqualsWithIgnoreCase` and `ContainsWithIgnoreCase`.